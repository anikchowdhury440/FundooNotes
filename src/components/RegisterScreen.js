import React from 'react';
import Register from './Register';

const RegisterScreen = ({navigation}) => {
    return (
        <Register navigation = {navigation}/>
    )
}

export default RegisterScreen