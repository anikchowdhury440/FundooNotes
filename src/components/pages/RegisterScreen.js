import React from 'react';
import Register from '../authentication/Register';

const RegisterScreen = ({navigation}) => {
    return (
        <Register navigation = {navigation}/>
    )
}

export default RegisterScreen