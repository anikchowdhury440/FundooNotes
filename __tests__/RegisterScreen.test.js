import React from 'react';
import {shallow} from 'enzyme';
import RegisterScreen from '../src/components/RegisterScreen';

describe('test RegisterScreen', () => {
    it('should match to snapshot', () => {
        const component = shallow(<RegisterScreen/>)
        expect(component).toMatchSnapshot();
    })
})