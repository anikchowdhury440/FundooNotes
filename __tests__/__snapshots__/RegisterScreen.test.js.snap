// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test RegisterScreen should match to snapshot 1`] = `ShallowWrapper {}`;
